src/toBeReviewed.js

```diff
@@ -0,0 +1,26 @@
+var animalData = [
+  {
+    id: 1,
+    title: "hippo",
+    faveFood: "carrots",
+  },
+  {
+    id: 2,
+    title: "Cat",
+    faveFood: "carrots",
+  },
+  {
+    id: 3,
+    title: "ducks",
+    faveFood: "breadcrumbs",
+  },
+];
+
+exports.findAnimal = function () {
+  for (x = 0; x < 3; x++) {
+    if ((animalData[x].title = args[0])) {
+      var answer = animalData[x].faveFood;
```

Comment:
It is better to return the animal object instead of its particular property to make the function more useful and matching with its naming.